SpatialStructure-4/SpatialStructure-4.ifc

```diff
@@ -2,36 +2,36 @@ ISO-10303-21;
 HEADER;
 FILE_DESCRIPTION(('ViewDefinition [Ifc4X3NotAssigned]'),'2;1');
 FILE_NAME(
-/* name */ 'SpatialStructure-4.ifc',
-/* time_stamp */ '2021-01-21T14:36:20',
+/* name */ 'SpatialStructure_LC.ifc',
+/* time_stamp */ '2021-03-18T10:56:35',
```

Comment:
Well is it `_LC` or `-4`?

Reply:
It is -4. Now corrected. I added an additional level. We have added a technical building for the railway signaling equipment